opennms-base-assembly/src/main/filtered/bin/install

```diff
@@ -6,12 +6,18 @@ OPENNMS_BINDIR="${install.bin.dir}"
 RUNAS="root"
 
 ULIMIT=`which ulimit 2>/dev/null`
```

Comment:
Just to verify:
```
meridian-sh ~ # cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)
meridian-sh ~ # bash -x test.sh
++ which ulimit
+ ULIMIT=
+ echo

meridian-sh ~ # cat test.sh
ULIMIT=`which ulimit 2>/dev/null`
echo $ULIMIT
```

this is empty? you can execute it and it gives 

```
meridian-sh ~ # ulimit
unlimited
```
This is intended?